### PR TITLE
Readings

### PR DESCRIPTION
**Pull request by gprocell927**
@martensonbj
This PR creates a reading that is recorded in firebase. Before connecting with firebase, the readings recorded in the redux store.  Once firebase was connected, the readings stopped recording in the store. I tried to connect mapStateToProps and mapDispatchToProps in components/ReadingsList.js , but that did not change anything. Have I missed a store connection? Please let me know if I need to specify anything else.

### CODE REVIEW

**Comment by martensonbj** [comment]
I'm pulling in @brittanystoroz for an extra set of eyes too

File: anesthesia-monitor/src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import firebase, { reference } from '../utils/firebaseClient'
```
Comment (brittanydionigi) [inline anesthesia-monitor/src/actions/index.js]: Where is this `../utils/firebaseClient` file? I don't see it in the repo anywhere. It's tough to know what `reference` is, which makes it hard to debug. Is it a file you created or something pulled directly from the firebase library?

File: anesthesia-monitor/src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    })
+  }
+}
+
+export const addReading = (reading) => {
+  // create new record in firebase from info
+    // if successful, update the redux store
+    // if failure, show an error message
+  return dispatch => {
+    const id = reference.push(reading).path.o[1]
```
Comment (brittanydionigi) [inline anesthesia-monitor/src/actions/index.js:19]: Can you give me a little context on what `.path.o[1]` is supposed to be? From a quick glance, the thing that looks a bit off here is that you're not actually waiting for a successful firebase transaction to finish before you start dispatching an action.  I should really look more like something along these lines: 

```javascript
    reference
      .push('readings', reading)
      .then(() => {
        dispatch({
             type: 'ADD_READING',
             info: reading,
             id
        })
     })
```

File: anesthesia-monitor/src/actions/index.js (hunk 1 above)
Comment (brittanydionigi) [reply anesthesia-monitor/src/actions/index.js]: jk, I see it was a file added in this PR

File: anesthesia-monitor/src/index.js [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+import reducers from './reducers';
+import App from './components/App';
 import './index.css';
 
-ReactDOM.render(
-  <App />,
+
+const createStoreWithMiddleware = applyMiddleware(reduxThunk)(createStore);
+
+let store = createStore(reducers, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
```
Comment (brittanydionigi) [inline anesthesia-monitor/src/index.js:21]: The way you're creating your store looks a little bit off as well. It looks like if you're trying to use `react-redux-firebase` with `redux-thunk`, you need to add some extra arguments to your store creation.  See the documentation [here](https://github.com/prescottprue/react-redux-firebase#using-with-redux-thunk).

**Pull request merged**